src/main/webapp/js/adminActivityLog.es6

```diff
@@ -6,6 +6,23 @@ $(document).ready(() => {
     $('#filterReference').toggle();
     bindBackToTopButtons('.back-to-top-left, .back-to-top-right');
     highlightKeywordsInLogMessages();
+
+    $('#toggleReferenceLink').on('click', (e) => {
+        e.preventDefault();
```

Comment:
Let's deal with this before moving forward, this will cause us more problems (and I think it already does) when we try to migrate to ES6 and beyond